### ISSUE TITLE
Tag for 1.2.2 is missing

### ISSUE THREAD

**Issue by valeriomazzeo** [muZZkat/NMRangeSlider#69]
as per title, updating / installing from cocoapods still picks up 1.2.1

**Comment by fmaylinch** [comment]
Yep, neither tag 1.2.2 was created on github nor this version was uploaded to cocoapods.

You can use this podfile dependency to get the last version (last commit so far):

`pod 'NMRangeSlider', :git => 'https://github.com/muZZkat/NMRangeSlider.git', :commit => '56e8676' # version 1.2.2 not created yet`
